### PR TITLE
Set up profile, and add blog post, some style fixes for mobile too

### PR DESCRIPTION
**Pull request by BenjaminWalsh**
- Added my own profile info and page
- Added my blog post
- Uploaded my compressed images
- Updated job titles for existing authors
- Better spacing around body text on mobile and images default to centre alignment

### CODE REVIEW

File: _data/people.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 - author: jonathan_wilson
   name: Jonathan Wilson
-  title: Technology Director
+  title: Chief Technology Officer
```
Comment (michaelbinks) [inline _data/people.yml]:
```suggestion
  title: Chief Product Officer
```

Comment (BenjaminWalsh) [reply _data/people.yml]: Good catch, thanks :)

**Comment by BenjaminWalsh** [comment]
@michaelbinks github/pages checks still havent been reported, so I can't merge without using my admin. Should we fix this before or after I deploy these changes? And when I say deploy, is this automatic when merging with master, or is there a bamboo plan for this?

**Comment by michaelbinks** [comment]
@BenjaminWalsh WTF!!! I was showing Matt how it kept failing and it merged!

**Pull request merged**